### PR TITLE
update conferences to move React Rally 23 to previous and add upcoming conferences

### PR DESCRIPTION
**Pull request by M0nica**
- update conferences to move React Rally 23 to previous 
- add React Alicante, reactjsday 2023,  React Live, React Brussels 2023 and Redwood Conf to upcoming

### CODE REVIEW

**Review by AbdullahWins — COMMENTED** [review]
well done!

**Comment by AymenBenAmor** [comment]
@M0nica Thank you for adding our conference (React Brussels to the list). I had a PR for it but I'll close it :) 

thank you

**Comment by AymenBenAmor** [comment]
Hi,

Can we do something to get this PR merged? We really would love to have our React conference (https://www.react.brussels/) displayed on the website...

File: src/content/community/conferences.md [hunk 1]
```diff
@@ -10,21 +10,42 @@ Do you know of a local React.js conference? Add it here! (Please keep the list c
...
-[Website](https://www.reactrally.com/) - [Twitter](https://twitter.com/ReactRally) - [Instagram](https://www.instagram.com/reactrally/)
+[Website](https://www.redwoodjsconf.com/) - [Twitter](https://twitter.com/redwoodjs)
+
+### React Alicante 2023 {/*react-alicante-2023*/}
+September 28 - 30, 2023. Alicante, Spain
+
+[Website](https://reactalicante.es/) - [Twitter](https://twitter.com/reactalicante)
+
+### React Live 2023 {/*react-live-2023*/}
+September 29, 2023. Amsterdam, Nederland
```
Comment (harish-sethuraman) [inline src/content/community/conferences.md:24]:
```suggestion
September 29, 2023. Amsterdam, Netherlands
```

changing it for consistency

**Comment by AymenBenAmor** [comment]
Thank you so much :)

**Pull request merged**